.github/workflows/dapr-test.yml

```diff
@@ -334,12 +334,12 @@ jobs:
       - name: Build Dapr and its Docker images
         if: env.TEST_PREFIX != ''
         run: |
-          make build
+          make build -j4
```

Comment:
I believe our Actions runners have 2 cores, so I wouldn't set to more than 2 parallel jobs.

Reply:
it seems that I do not have access to the runner's details. Thank you for the information！

Reply:
Also note that in _most_ cases, the images are cached, so they shouldn't be rebuilt... unless something changes :)

Reply:
Thanks！do you have any idea about how to speed up the build process? Feel free to comment here or in the issue.

Reply:
I think the images that take the longest are the Java and Python ones, IIRC? When they are not cached

Reply:
Let's take a look at a recent E2E test: https://github.com/dapr/dapr/actions/runs/7039695179

The longest thing (aside from test) is actually retrieving the Go dependencies, which is very slow especially on Windows (lots of disk I/O with lots of small files). This is a problem with all our actions since we have A LOT of dependencies. I don't think there's much we can do there to speed that up sadly :(